ghost/core/core/server/api/endpoints/pages.js

```diff
@@ -238,5 +238,31 @@ module.exports = {
         query(frame) {
             return models.Post.destroy({...frame.options, require: true});
         }
+    },
+
+    copy: {
+        statusCode: 201,
+        headers: {},
+        options: [
+            'id',
+            'formats'
+        ],
+        validation: {
+            id: {
+                required: true
+            }
+        },
+        permissions: {
+            docName: 'posts',
+            method: 'add'
+        },
+        async query(frame) {
+            return postsService.copyPost(frame)
+                .then((model) => {
+                    this.headers.cacheInvalidate = false;
```

Comment:
we don't need this declaration here if we also remove the `headers: {},` declaration above. in this case the code just becomes `return postsService.copyPost(frame);`

Reply:
Fixed in https://github.com/TryGhost/Ghost/pull/16767/commits/bd8dc9037be19589336dd0bb834ab58ccfe63206